ci-operator/step-registry/osd/create/create/osd-create-create-ref.yaml

```diff
@@ -7,6 +7,7 @@ ref:
     requests:
       cpu: 100m
       memory: 300Mi
+  timeout: 3h0m0s
```

Comment:
Why 3 hours here? Could we make this 1h?

Reply:
@asergienk could you make it 2hrs.

Reply:
done, it's 2hrs now